bindings/python/pymongoarrow/types.py

```diff
@@ -288,7 +288,10 @@ def _normalize_typeid(typeid, field_name):
             fields.append((sub_field_name, _normalize_typeid(sub_typeid, sub_field_name)))
         return struct(fields)
     if isinstance(typeid, list):
-        return list_(_normalize_typeid(type(typeid[0]), "0"))
+        if len(typeid) == 0 or len(typeid) > 1:
+            msg = "Please provide a one datatype or struct in case of multiple datatypes in a list"
```

Comment:
Suggest: `f"list field in schema must contain exactly one element, not {len(typeid)}"`

Reply:
Accept